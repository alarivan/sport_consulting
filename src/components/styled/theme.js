import { lighten } from "polished";

const primaryColor = "#1171b7";

const theme = {
  primaryColor,
  primaryColorHover: lighten(0.1, primaryColor),
  borderColor: "#ccc",
  textColor: "#444444"
};

export default theme;
