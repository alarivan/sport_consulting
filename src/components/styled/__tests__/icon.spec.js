import React from "react"
import { render } from "@testing-library/react"
import Icon from "../Icon.css.js"
import theme from "../theme"

describe("Button", () => {
  it("renders correctly", () => {
    const { asFragment } = render(<Icon />)

    expect(asFragment()).toMatchSnapshot()
  })

  it("renders correctly with theme", () => {
    const { asFragment, container } = render(<Icon theme={theme} />)

    expect(container.firstChild).toHaveStyleRule("fill", theme.textColor)

    expect(asFragment()).toMatchSnapshot()
  })
})
