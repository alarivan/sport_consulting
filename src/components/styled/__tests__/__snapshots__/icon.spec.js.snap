// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<DocumentFragment>
  .c0 {
  display: block;
  width: 1rem;
  height: 1rem;
}

<svg
    class="c0"
  />
</DocumentFragment>
`;

exports[`Button renders correctly with theme 1`] = `
<DocumentFragment>
  .c0 {
  display: block;
  fill: #444444;
  width: 1rem;
  height: 1rem;
}

<svg
    class="c0"
  />
</DocumentFragment>
`;
